qunicorn_core/api/job_api/job_view.py

```diff
@@ -115,5 +115,21 @@ class JobPauseView(MethodView):
     def post(self, body, job_id: str):
         """TBD: Pause a job via id."""
         logging.info("Request: pause job")
-
         return jsonify(jobmanager_service.pause_job_by_id(job_id))
+
+
+@JOBMANAGER_API.route("/<string:deployment_id>/jobs")
+class DeploymentDetailJobView(MethodView):
+    """API endpoint for jobs of a specific deployment."""
+
+    @JOBMANAGER_API.response(HTTPStatus.OK, JobResponseDtoSchema(many=True))
+    def get(self, deployment_id: str):
+        """Get the details of all jobs with a specific deployment id."""
+        logging.info("Request: get jobs with deployment id")
+        return jsonify(jobmanager_service.get_jobs_by_deployment_id(deployment_id)), 200
```

Comment:
Dieses ", 200" würde ich weglassen. dafür haben wir ja extra den Code "HTTPStatus.OK" in der Anotation.